.chloggen/add-managed-state.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add the `managementState` to the TempoStack spec. It enables/disables the reconciliation in the operator
```

Comment:
`Add  `.spec.managedState` to `TempoStack`. It allows disabling reconciliation of the `TempoStack` CR.

Reply:
Let's document the behaviour of validation and defaulting webhook for unmanaged state.

Reply:
@pavolloffay should I disable the validation and defaulting? That is not currently done

Reply:
I think we should keep it enabled for now